zio-http/src/main/scala/zhttp/http/middleware/Web.scala

```diff
@@ -65,25 +67,13 @@ private[zhttp] trait Web extends Cors with Csrf with Auth {
    * Sets cookie in response headers
    */
   def addCookie(cookie: Cookie): HttpMiddleware[Any, Nothing] =
```

Comment:
`addCookie` should move to `HeaderModifiers`

Reply:
withSetCookie is already there
but we need to maintain consistency with `addCookieZIO` hence added `addCookie`